client/js/shout.js

```diff
@@ -197,6 +197,23 @@ $(function() {
 				data.msg
 			]);
 
+		chan.find(".messages .text:not(.processed)")
+			.each(function() {
+				var html = $(this).html();
+				$(this)
+					.addClass("processed")
+					.html(html.replace(/(\W)(#[\w\d-.]{0,200})/ig,
```

Comment:
* I couldn't find a network where &channel is working. Are you sure that this is supported by shout's backend?
* Actually, I do think so. Test it on the shout demo network for example. `/join #` works just fine.
* I agree.